### DISCUSSION TITLE
wine: launching multiple wine applications crashes previous ones

### DISCUSSION THREAD

**Discussion by Bluey26** [netblue30/firejail#6138] [Q&A] [answered]
Hello.

This seems to be the place for questions, i am having troubles to start and close apps using `firejail` and `wine`

I expect that opening app `app01.exe` and `app02.exe` using `firejail wine ` would not 'interfere' between each other.

What i have is: i open both apps. If i close one of them, the other one 'crashes' and becomes unusable, it stops responding, without apparent reason.
I have tried different  firejail profiles, and it also happens.

I would like to run both apps and avoid that, closing "app02" and that "app01" still works. I do not know if this is possible or if this is a bug.
I also have the following problem: If i open an app with the option: `--net=none` and then i open another app, with firejail (but without the` --net=none` argument), the 2nd app does not have internet access. Instead, i have to open the 2nd app(internet access) and then the other one(`--net=none` one). How can i make them 'independent'? So i can open them in any order, closing and open one and let the internet app work normally.

I would like to do this without recurring to use different wineprefixes, i have not tried that yet, but its undesirable(everything is in the same wineprefix).

Info:
Firejail version =  0.9.72
OS: Arch Linux
wine version = 8.21

Thanks in advance.

Relates to:

* #6084
* #6826

**Comment by Bluey26** [comment]
any suggestions on how could i do this?

**Reply by rusty-snake** [reply, marked as answer]
Since you want to have a different security policy for app01 and app02, you should consider to run them in two sandboxes with a different WINE_PREFIX.

**Reply by Bluey26** [reply]
I decided to make the new prefix for app1, i executed it using the new prefix path and after that, launching app2 seems to work without issues. Thank you
